substrate/frame/grandpa/src/mock.rs

```diff
@@ -195,7 +195,7 @@ impl pallet_staking::Config for Test {
 	type SessionInterface = Self;
 	type UnixTime = pallet_timestamp::Pallet<Test>;
 	type EraPayout = pallet_staking::ConvertCurve<RewardCurve>;
-	type MaxNominatorRewardedPerValidator = ConstU32<64>;
```

Comment:
can't wait to use `derive_impl` for staking as well, it is one of those pallet that is used in a couple of mocks.